src/App.js

```diff
@@ -3,13 +3,25 @@ import Box from '@mui/material/Box'
 import Toolbar from '@mui/material/Toolbar'
 import Typography from '@mui/material/Typography'
 import { LoginPage } from './pages/login-page/LoginPage.js';
-import { useLocation, Routes, Route,} from "react-router-dom"
+import { useLocation, Routes, Route,} from "react-router-dom";
+import { TransactionHistoryPage } from './pages/transaction-history-page/TransactionHistory.js';
+import { ResponsiveDrawer } from './components/Drawer.js';
+import {useEffect} from 'react'
 
 export default function App() {
 
+  useEffect(() => {
+    document.body.style.height = '100vh';
+    const root = document.getElementById('root');
+    root.style.height = 'calc(100% - 10rem)'
+
+    return () => {
+    };
+  }, []);
+
   let isLogin = true;
```

Comment:
Set the variable to false, and down below just check if the pathname is the needed one and then set it to true, no need for else statement.

```suggestion
  let isLogin = false;
```